<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow, QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>621</width>
      <height>761</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow, QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
}
</string>
    </property>
    <widget class="QFrame" name="frame_menu">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>361</width>
       <height>431</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
    background-color: white;
    border-radius: 20px;
    padding: 20px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_adminprefencemenu">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    background-color: #7f7f7f;
    color: white;
    font-weight: bold;
    font-size: 18px;
    padding: 6px 12px;
    border-radius: 8px;
}


</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>CRM-Admin Preference Menu</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_mentorinterview">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #b30000;
    color: white;
    border-radius: 10px;
    padding: 8px 20px;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #8b0000;
}

QPushButton:hover {
    background-color: #e60000;
    border: 2px solid #ff0000;
}

</string>
        </property>
        <property name="text">
         <string>Mentor Interview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_applications">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #b30000;
    color: white;
    border-radius: 10px;
    padding: 8px 20px;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #8b0000;
}

QPushButton:hover {
    background-color: #e60000;
    border: 2px solid #ff0000;
}

</string>
        </property>
        <property name="text">
         <string> Applications</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/categories.png</normaloff>../images/categories.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_interviews">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #b30000;
    color: white;
    border-radius: 10px;
    padding: 8px 20px;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #8b0000;
}

QPushButton:hover {
    background-color: #e60000;
    border: 2px solid #ff0000;
}
</string>
        </property>
        <property name="text">
         <string>Interviews</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_adminmenu">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #b30000;
    color: white;
    border-radius: 10px;
    padding: 8px 20px;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #8b0000;
}

QPushButton:hover {
    background-color: #e60000;
    border: 2px solid #ff0000;
}
</string>
        </property>
        <property name="text">
         <string>Admin Menu</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/home.png</normaloff>../images/home.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #b30000;
    color: white;
    border-radius: 10px;
    padding: 8px 20px;
    font-weight: bold;
    font-size: 14px;
    border: 2px solid #8b0000;
}

QPushButton:hover {
    background-color: #e60000;
    border: 2px solid #ff0000;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/logout.png</normaloff>../images/logout.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="comboBox_language">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>720</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox#comboBox_language {
    background-color: #ffffff;
    color: #2c5364;  /* Tema ile uyumlu koyu mavi */
    border: 2px solid #2c5364;
    border-radius: 5px;
    padding: 4px;
    min-width: 10px;
    font-weight: bold;
}
QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: #2c5364;
    selection-background-color: #203a43;
    selection-color: white;
}
</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Türkçe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dutch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deutsch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Русский</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Español</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>العربية</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>简体中文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日本語</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_language">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>720</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_language {
    color: white;
    font-weight: bold;
    font-size: 6pt;
}
</string>
     </property>
     <property name="text">
      <string>Languages</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_logo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/logo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
